style.scss

```diff
@@ -18,10 +18,11 @@ $color-4: #7f8cff;
   body {
     font-family: "Poppins", sans-serif;
     font-style: normal;
-    max-width: 23.75rem;
-    display: flex;
-    justify-content: center;
-    flex-direction: column;
+    // max-width: 380px;
+    // margin: 0 auto;
+    // display: flex;
+    // justify-content: center;
+    // flex-direction: column;
```

Comment:
- [ ] Hey @tsheporamantso, thanks for following the comments other reviewers said, you did well on making those changes, but please do not make your codebase look unclean by leaving commented lines of codes, it is always recommended to remove them as it makes your codebase more professional and clean, I would suggest you fix this issue and all is good afterward.